File: report/src/main/java/com/vmlens/report/runelementtype/LockOperation.java
package com.vmlens.report.runelementtype;

public enum LockOperation {

    LOCK_ENTER("Enter"),
    LOCK_EXIT("Exit") ,
    CONDITION_WAIT_ENTER("Wait Enter"),
    CONDITION_WAIT_EXIT("Wait Exit");

    private final String text;

    LockOperation(String text) {
        this.text = text;
    }

    public String text() {
        return text;
    }
}

File: sync-bug/src/main/scala/com/anarsoft/race/detection/report/element/runelementtype/LockOperation.scala
package com.anarsoft.race.detection.report.element.runelementtype

enum LockOperation(val text: String) {
  case LOCK_ENTER extends LockOperation("Enter")
  case LOCK_EXIT extends LockOperation("Exit")
  case CONDITION_WAIT_ENTER extends LockOperation("Wait Enter")
  case CONDITION_WAIT_EXIT extends LockOperation("Wait Exit")
}